azure-pipelines.yml

```diff
@@ -202,12 +202,12 @@ jobs:
     displayName: Copy EndpointManager Launcher to dsc folder
     enabled: True
     inputs:
-      SourceFolder: src/EndpointManagerLauncher/bin/${{ parameters.BuildConfiguration }}/net6.0/win-x64/publish/
+      SourceFolder: src/EndpointManagerLauncher/bin/${{ parameters.BuildConfiguration }}/net7.0/win-x64/publish/
```

Comment:
any reason we dont have `netx.x` as a param as well so we dont have to change it here every time?

Reply:
@bfoley13 thanks for this good suggestion, I will check it out.